### DISCUSSION TITLE
Reached artifact limit?

### DISCUSSION THREAD

**Discussion by davidjdixon** [community/community#27095] [Actions]
<p>We’ve started seeing failures in a 1/2 hourly CI process that created 4 artifacts (each with a single HTML file with unit test results).</p>
<p>What we are seeing are the following messages: </p>
<pre><code>##[error]Cannot upload artifacts. You may have reached your capacity.
##[error]Exit code 1 returned from process: file name '/home/runner/runners/2.160.2/bin/Runner.PluginHost', arguments 'action "GitHub.Runner.Plugins.Artifact.PublishArtifact, Runner.Plugins"'.
</code></pre>
<p>I cannot see any information on artifacts limits in the <a href="https://help.github.com/en/actions/automating-your-workflow-with-github-actions/persisting-workflow-data-using-artifacts" rel="nofollow noopener">artifacts page </a>or the section on <a href="https://help.github.com/en/actions/automating-your-workflow-with-github-actions/about-github-actions#usage-limits" rel="nofollow noopener">usage limits</a>. But I would not expect any limit to be reached under such circumstances since these are perfectly reasonable artifacts to create from a CI run.</p>
<p>This is obviously a large concern over usage limits, especially as we’re on a paid teams plan. Can anyone from Github comment?</p>

**Comment by pixelmatrix** [comment]
<p>I saw this same error earlier too, and then my build got stuck. Seems like there’s some kind of outage with actions right now.</p>

**Comment by jperl** [comment]
<p>Also running into this issue here <a href="https://github.com/qawolf/examples/commit/f3e2f49400e0f60d4bf495d5ccd55a60bd2cc7c1/checks" rel="nofollow noopener">https://github.com/qawolf/examples/commit/f3e2f49400e0f60d4bf495d5ccd55a60bd2cc7c1/checks</a></p>

**Comment by davidjdixon** [comment]
<p>No problem. Done.</p>

**Comment by Hexeption** [comment]
<p>I’ve had this happen to me too.</p>

**Comment by mscoutermarsh** [comment]


**Reply by cdjambovyourbteam** [reply]
How exactly ? Thanks

**Comment by Trimud** [comment]
<p>You need to add new step:</p>
<pre><code>- name: Create artifact dir
        run: |
          mkdir -partifacts
</code></pre>

**Comment by sinoru** [comment]
<p>You means fixes this bug? Or you updates <a href="https://help.github.com/en/github/setting-up-and-managing-billing-and-payments-on-github/about-billing-for-github-actions" rel="nofollow noopener">docs</a>? Even that, Currently there are no way to remove artifacts to reclaim the storage.</p>
<p>I can’t use workflow to create archives right now because of that. Could you look at it asap?</p>

**Comment by egordorichev** [comment]
<p>Stumbled into this issue today, super annoying… </p>
<p>And I didn’t find any info on how to fix or why this is happening.</p>

**Comment by egordorichev** [comment]
<p>Interesting, so I figured out, that under <a href="https://github.com/settings/billing" rel="nofollow noopener">https://github.com/settings/billing</a> it says how much artifact space you have, it’s 1GB in my case, and I ran out of it. But I don’t see a way to delete artifacts…</p>

**Comment by IvanBioRender** [comment]
<p>Same here. Our team was already a bit over the limit, so I’m a bit confused why it only started giving errors now, after we’ve already been allowed to go over the limit and get charged.</p>
<p>I’m also very confused at how the <a href="https://github.com/actions/cache/" rel="nofollow noopener">cache action</a> plays into this, especially since it has <a href="https://github.com/actions/cache/#cache-limits" rel="nofollow noopener">it’s own limit</a>.</p>
<p>We’ve been caching test reports and failure screenshots, which are all farily small, so it’s strange that we hit over 2 GB of storage used, which is why I think the cache action (which caches our <code>node_modules</code>) might contribute to that. But again, it has it’s own limits, so I have no clue.</p>

**Comment by bfloch** [comment]
<p>I am also seeing storage going up but no way to delete old artifacts.</p>
<p>Is this WIP?</p>

**Comment by egordorichev** [comment]
<p>For anyone still looking for a solution, there is a workaround. It might seem a bit dirty, but it works. Instead of pushing your artifacts using the upload-artifact, just push it to a special private repo. Worked just fine for me :slight_smile:</p>

**Comment by kylemath** [comment]
<p>we are still getting this same problem on our private repo <a href="http://github.com/kylemath/EEGEdu" rel="nofollow noopener">http://github.com/kylemath/EEGEdu</a></p>
<p>Please advise how to remedy</p>
